libs/sdk-backend-bear/src/backend/workspace/metadata/index.ts

```diff
@@ -277,7 +284,8 @@ export class BearWorkspaceMetadata implements IWorkspaceMetadata {
             content: { formOf, default: defaultDisplayForm },
         } = displayFormDetails;
         const ref: UriRef = uriRef(uri);
-        const isDefaultDf = !!(defaultDisplayForm === 1);
+        const isDefaultDf =
+            defaultDisplayForm !== undefined ? !!(defaultDisplayForm === 1) : defaultDisplayForm;
```

Comment:
why `defaultDisplayForm === 1` is not sufficient?

Reply:
Because `defaultDisplayForm` can be `undefine`, we just need to check it when it has value.

Reply:
When it's undefined, the condition returns false, which should be a correct value or not?

Reply:
If the attribute has only a displayForm, the value will be `undefined` and we don't need to check it.

Reply:
@fixed with `defaultDisplayForm === 1` condition